internal/runners/config/get.go

```diff
@@ -30,7 +29,7 @@ func (g *Get) Run(params GetParams) error {
 
 	value, err := configMediator.GetRule(params.Key.String()).GetEvent(value)
 	if err != nil {
-		logging.Error("Could not execute additional logic on config get, err: %w", err)
+		return locale.WrapError(err, "err_config_get_event", "Could not execute config get event")
```

Comment:
```suggestion
		return locale.WrapError(err, "err_config_get_event", "Could not retrieve config value, if this continues to happen please contact support.")
```

Always write these with the end-user in mind.